packages/zoltan2/src/problems/Zoltan2_ColoringProblem.hpp

```diff
@@ -201,6 +209,22 @@ void ColoringProblem<Adapter>::solve(bool newData)
       AlgSerialGreedy<Adapter> alg(this->graphModel_, this->params_,
                                    this->env_, this->comm_);
       alg.color(this->solution_);
+  } 
+  else if (method.compare("Hybrid") == 0)
+  {
+      //new stuff using Kokkos and Gebremedhin-Manne-Boman framework for hybrid architectures
+      //this->inputAdapter_ is the adapter passed to this problem
+      printf("Do the hybrid algorithm\n"); 
```

Comment:
Is this a debug printf? The deafult behavior should be silent.

Reply:
I have removed these printfs. They were primarily for debugging.